### ISSUE TITLE
Remove final layer from symbol

### ISSUE THREAD

**Issue by erogol** [dmlc/mxnet#1004]
I read a json file network structure and I want to remove the final softmax layer. Is it viable or do I need to redefine the network?

**Comment by tqchen** [comment]
You can use getinternals to retrieve the internals of a symbol, see last block of the notebook
https://github.com/dmlc/mxnet/blob/master/example/notebooks/predict-with-pretrained-model.ipynb

**Comment by r0mer0m** [comment]
The referenced notebook is no longer available. Can you post the explicit solution?
